### PR TITLE
[Various component] Add high contrast to design language

### PR DESCRIPTION
**Pull request by dleroux**
### WHY are these changes introduced?

Fixes minor issues with Popover, Card
Applies high contrast to the indicator, checkable button, option list

**Bulk actions and checkable button**
Screen Shot 2020-02-27 at 4 08 21 PM

**indicator**
Screen Shot 2020-02-27 at 3 39 53 PM

**popover**
Screen Shot 2020-02-27 at 2 51 57 PM

**card**
Screen Shot 2020-02-27 at 2 50 53 PM

**option list**
Screen Shot 2020-02-27 at 2 47 18 PM

## 

### How to 🎩

🖥 [Local development instructions](https://github.com/Shopify/polaris-react/blob/master/README.md#development)
🗒 [General tophatting guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md)
📄 [Changelog guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Versioning%20and%20changelog.md)

<details>
<summary>Copy-paste this code in <code>playground/Playground.tsx</code>:</summary>

```jsx
import React from 'react';
import {Page} from '../src';

export function Playground() {
  return (
    <Page title="Playground">
      {/* Add the code you want to test in here */}
    </Page>
  );
}

```

</details>

### 🎩 checklist

* [ ] Tested on [mobile](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md#cross-browser-testing)
* [x] Tested on [multiple browsers](https://help.shopify.com/en/manual/intro-to-shopify/shopify-admin/supported-browsers)
* [ ] Tested for [accessibility](https://github.com/Shopify/polaris-react/blob/master/documentation/Accessibility%20testing.md)
* [ ] Updated the component's `README.md` with documentation changes
* [ ] [Tophatted documentation](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting%20documentation.md) changes in the style guide
* [ ] For visual design changes, pinged one of @ HYPD, @ mirualves, @ sarahill, or @ ry5n to update the Polaris UI kit

### CODE REVIEW

File: src/components/OptionList/components/Checkbox/Checkbox.scss [hunk 1]
```diff
@@ -85,7 +85,6 @@
         @include control-backdrop(active, $global-theming: true);
       }
       ~ .Icon {
-        transition-timing-function: var(--p-ease-in);
```
Comment (dleroux) [inline src/components/OptionList/components/Checkbox/Checkbox.scss]: this was an oversight.

File: src/components/Popover/Popover.scss [hunk 2]
```diff
@@ -58,9 +58,7 @@ $content-max-width: rem(400px);
   overflow: hidden;
   background-color: var(--p-surface, color('white'));
   border-radius: var(--p-border-radius-wide, border-radius());
-  @media (-ms-high-contrast: active) {
-    box-shadow: inset 0 0 0 border-width(base) ms-high-contrast-color('text');
-  }
+  outline: 1px solid transparent;
```
Comment (dleroux) [inline src/components/Popover/Popover.scss]: simplified with this. box-shadow had no effect.

File: src/components/Card/Card.scss [hunk 3]
```diff
@@ -4,6 +4,7 @@
   overflow: var(--p-override-visible, hidden);
   background-color: var(--p-surface, color('white'));
   box-shadow: var(--p-card-shadow, shadow());
+  outline: border-width(base) solid transparent;
```
Comment (dleroux) [inline src/components/Card/Card.scss]: box-shadow had no effect. Could be a chromium edge update

**Comment by chloerice** [comment]
(Removed myself as I still need to set up a virtual machine)

**Pull request merged**